apps/couchdb.go

```diff
@@ -61,3 +62,122 @@ func (r MetricsReceiverCouchdb) Pipelines() []otel.Pipeline {
 func init() {
 	confgenerator.MetricsReceiverTypes.RegisterType(func() confgenerator.Component { return &MetricsReceiverCouchdb{} })
 }
+
+type LoggingProcessorCouchdbError struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+}
+
+func (LoggingProcessorCouchdbError) Type() string {
+	return "couchdb"
+}
+
+func (p LoggingProcessorCouchdbError) Components(tag string, uid string) []fluentbit.Component {
+	c := confgenerator.LoggingProcessorParseMultilineRegex{
+		LoggingProcessorParseRegexComplex: confgenerator.LoggingProcessorParseRegexComplex{
+			Parsers: []confgenerator.RegexParser{
+				{
+					// Format https://github.com/apache/couchdb/blob/main/src/couch_log/src/couch_log_writer_syslog.erl#L72
+					// Sample line: [notice] 2021-12-02T23:36:42.555157Z nonode@nohost <0.17165.1> a5f585a0d3 localhost:5984 127.0.0.1 otelu PUT /oteld 201 ok 16
+					Regex: `^\[(?<severity>\w*)\] (?<timestamp>[\d\-\.:TZ]+) (?<node>\w+)@(?<host>[^\s]+) \<(?<pid>[^ ]*)\> [\w-]+ (?<http_request_serverIp>[^ ]*) (?<http_request_remoteIp>[^ ]*) (?<remote_user>[^ ]*) (?<http_request_requestMethod>[^ ]*) (?<path>[^ ]*) (?<http_request_status>[^ ]*) (?<message>[^ ]*) (?<http_request_responseSize>[\d]*)$`,
```

Comment:
I don't think what you've captured here as "message" would be useful. Maybe capture remote_user through the end as message. You can still capture subfields inside (nested capture groups), but this'll leave a useful message in the logs explorer as the "message".

Reply:
updated message to capture remote user to end. Renamed old message to response_message.